src/main/kotlin/apply/utils/ExcelGenerator.kt

```diff
@@ -3,58 +3,58 @@ package apply.utils
 
-            styleHeaders(workbook, sheet, headerTitles)
-            fillData(sheet, rows, headerTitles.size)
+        styleHeaders(workbook, sheet, headerTitles)
+        fillData(sheet, rows, headerTitles.size)
 
-            val out = ByteArrayOutputStream()
-            workbook.write(out)
-            workbook.close()
+        val out = ByteArrayOutputStream()
+        workbook.write(out)
+        workbook.close()
 
-            return ByteArrayInputStream(out.toByteArray())
-        }
+        return ByteArrayInputStream(out.toByteArray())
+    }
 
-        private fun styleHeaders(workbook: SXSSFWorkbook, sheet: SXSSFSheet, headerTitles: Array<String>) {
-            val headerFont = workbook.createFont()
-            headerFont.bold = true
+    private fun styleHeaders(workbook: SXSSFWorkbook, sheet: SXSSFSheet, headerTitles: Array<String>) {
+        val headerFont = workbook.createFont()
+        headerFont.bold = true
 
-            val headerCellStyle = workbook.createCellStyle()
-            headerCellStyle.setFont(headerFont)
-            headerCellStyle.fillBackgroundColor = IndexedColors.GREY_80_PERCENT.index
+        val headerCellStyle = workbook.createCellStyle()
+        headerCellStyle.setFont(headerFont)
+        headerCellStyle.fillBackgroundColor = IndexedColors.GREY_80_PERCENT.index
 
-            val headerRow = sheet.createRow(HEADER_ROW)
+        val headerRow = sheet.createRow(HEADER_ROW)
 
-            for (col in headerTitles.indices) {
-                val cell = headerRow.createCell(col)
-                cell.setCellValue(headerTitles[col])
-                cell.cellStyle = headerCellStyle
-            }
+        for (col in headerTitles.indices) {
+            val cell = headerRow.createCell(col)
+            cell.setCellValue(headerTitles[col])
+            cell.cellStyle = headerCellStyle
         }
+    }
 
-        private fun fillData(sheet: SXSSFSheet, rows: List<ExcelRow>, columnSize: Int) {
-            sheet.trackAllColumnsForAutoSizing()
+    private fun fillData(sheet: SXSSFSheet, rows: List<ExcelRow>, columnSize: Int) {
+        sheet.trackAllColumnsForAutoSizing()
 
-            rows.forEachIndexed { index, it ->
-                val row = sheet.createRow(index + 1)
-                val properties = it.getData()
-                properties.forEachIndexed { propertyIndex, property ->
-                    row.createCell(propertyIndex).setCellValue(property.toString())
-                }
+        rows.forEachIndexed { index, it ->
+            val row = sheet.createRow(index + 1)
+            val properties = it.data
+            properties.forEachIndexed { propertyIndex, property ->
+                row.createCell(propertyIndex).setCellValue(property)
             }
+        }
 
-            for (col in 0 until columnSize) {
-                sheet.autoSizeColumn(col)
-            }
+        for (col in 0 until columnSize) {
+            sheet.autoSizeColumn(col)
```

Comment:
`repeat()`를 찾아보고 그 기능을 사용해 보면 어떨까요?

```suggestion
        repeat(columnSize) {
            sheet.autoSizeColumn(it)
```

Reply:
코틀린은 syntax sugar 범벅이네요 ㅋㅋ